test/performance/odp_ipsec.c

```diff
@@ -632,17 +642,22 @@ run_measure_one(ipsec_args_t *cargs,
 		time_record_t *start,
 		time_record_t *end)
 {
+	int in_flight, pkts_allowed, num_out, num_enq, rc = 0;
+	const int max_in_flight = cargs->in_flight;
+	const int burst_size = cargs->burst_size;
+	odp_packet_t pkt[POOL_NUM_PKT];
```

Comment:
This could be made much smaller if moved down, i.e. similar change could be done here as was now done in the async case.

Reply:
Agreed. Will change in v4.